services/leecher/leecher/__main__.py

```diff
@@ -0,0 +1,9 @@
+import sys
+
+from .listener import ShotgridListener
```

Comment:
I think `__main__.py` does not support relative imports (not sure).

Reply:
So far the code hasn't complained :D